ODF-community.french.md

```diff
@@ -0,0 +1,526 @@
+ • Avons-nous impliqué les personnes qui vont devoir faire le travail ?<br/><br/>
+ • Auprès de qui devons nous chercher plus d'adhésion ou de soutien. 
+</td>
+</tr>
+<tr>
+<td><strong>Points clés</strong></td>
+</tr>
+<tr>
+<td>
+ • Representation de  différent types d'usagers<br/>
+ • Impacts et cas d'utilisation imprévus<br/>
+ • Risques et doutes non exprimés 
+</td>
+</tr>
+</table>
+
+<br/>
+
+Phase or activity: Launcer, Deployer, Cloturer <a name="launch"></a>
+--------------
+
+_page 10 sur 15_ 
+
+<table>
+<tr>
+<td colspan=2 width="66%"><strong>Étapes pour devenir ouvert</strong></td>
+<td><strong>Questions à poser</strong></td>
+</tr>
+<tr valign="top">
+<td rowspan=3 width=33%><strong>Commencer par envisager la fin</strong><br/>
+ • Demontrer l'alignment avec la stratégie, la mission, la culture et les
+veleurs de Red Hat<br/><br/>
+ • Souligner les étapes que vous avez suivie pour prendre votre décision de
+manière ouverte<br/><br/>
+ • Mettre en avant l'utilisation de ce cadre de décision ouverte <br/><br/>
+ • Dire aux collaborateurs ou trouver des informations détaillées<br/><br/>
+ • Monter comment le feedback a modelé la décision ou le projet<br/><br/>
+ • Expliquer comment fournir du feedback après le lancement<br/><br/>
+ • Signaler quand vous n'êtes pas complètement satisfait de la décision ou que
+vous savez que d'autres ne le seront pas. <br/><br/>
+ • Parteger votre feuille de route et vos critères pour revenir sur la décision<br/><br/>
+ • Rester en contact avec les personnes qui rejettent la décision
+</td>
+<td rowspan=3 width=33%>
+<strong>Ouvert par défaut</strong><br/>
+ • Réaffirmer les contraintes et les pré-requis pertinents <br/><br/>
+ • Partager les problèmes légaux, les rapports, et les
+questions de confidentialité concernés<br/><br/>
+ • Communiquer les critères de  succès et publier les métriques pertinentes<br/><br/>
+
+<strong>Contribuer en amont</strong><br/>
+ • Publier vos méthodes, les leçons apprises, les communiqués et les critères
+de décision dans les archives, pour que d'autres puissent étudier les anciennes
+décisions, comprendre pourquoi une décision a été prise et voir comment les
+leaders ont répondu à des problèmes similaires par le passé.<br/><br/>
+ • Proposer votre aide à d'autres personnes pour prendre des décisions ou
+choisir un outil de collaboration 
+</td>
+<td>
+ • Comment allons-nous suivre les listes de discussions et les autres canaux de
+feedback après le lancement ?<br/><br/>
+ • Si nous avons fait des version préliminaires, allons-nous continuer à faire
+des améliorations incrémentales basées sur le feedback ?<br/><br/>
+ • A quel point souhaitons nous faire des révisions sur la base du feedback ?<br/><br/>
+ • Quelle est la fenêtre temporelle raisonable pour l'afinage et les retours
+utilisateurs ?<br/><br/>
+ • Avons-nous sous-estimé un point important ? Comment le prendre en compte ?<br/><br/>
+ • Est-ce que la décision doit être revisée ?<br/><br/>
```

Comment:
both sounds good to me. `révisée` would be applied to a car that needs to be checked once every  2 years, so i liked this image. But ``réévaluée`` is fine.